services/motion/builtin/move_request.go

```diff
@@ -176,35 +183,42 @@ func (mr *moveRequest) deviatedFromPlan(ctx context.Context, plan motionplan.Pla
 // getTransientDetections returns a list of geometries as observed by the provided vision service and camera.
 // Depending on the caller, the geometries returned are either in their relative position
 // with respect to the base or in their absolute position with respect to the world.
+// TODO(RSDK-8340): Explore usefulness of pointcloud instead of spatialmath.Geometry.
```

Comment:
lets not commit this we already have the ticket for it

Reply:
here is another example of linking a ticket.
https://github.com/viamrobotics/rdk/blob/main/motionplan/check.go#L21

Is there a general rule of thumb to follow for when to do this?

Reply:
Imminent tickets or things we know we need to do, this is more of a speculative experiment.  I don't love TODO tickets in general but we do have a bunch of them you are right

Reply:
I understand now, thank you for explaining this to me!